rollup/l1/reader.go

```diff
@@ -16,7 +16,8 @@ import (
 
 const (
 	commitBatchEventName      = "CommitBatch"
-	revertBatchEventName      = "RevertBatch"
+	revertBatchV0EventName    = "RevertBatch"
+	revertBatchV7EventName    = "RevertBatch0"
```

Comment:
```suggestion
	revertBatchV7EventName    = "RevertBatchesInRange"
```

Reply:
The event is called `RevertBatch` and by printing all function identifiers I found that the overloaded event is called `RevertBatch0` in the ABI. 

https://github.com/scroll-tech/scroll-contracts/blob/f56af34b59b9c3a49e80ac09374df324346236a0/src/L1/rollup/IScrollChain.sol#L25

Reply:
Nice catch. It might be used carefully. e.g., not modifying this logic (alias ordering based on param diff) in the `abi` package